arches/app/datatypes/datatypes.py

```diff
@@ -161,6 +162,13 @@ def validate_from_rdf(self, value):
     def clean(self, tile, nodeid):
         if tile.data[nodeid] in ["", "''"]:
             tile.data[nodeid] = None
+        elif isinstance(tile.data[nodeid], dict):
+            for language_dict in tile.data[nodeid].values():
+                if language_dict["value"]:
```

Comment:
Assuming this key is present seemed legitimate.

Reply:
Yeah - that seems safe to me